### PR TITLE
Unconference

### PR DESCRIPTION
**Pull request by benfoxall**
This adds a page about the Unconference!

I'm not sure how/why it looks the way that it does, but I like it!

image

image

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -17,39 +17,43 @@
...
+  </title>
+  <meta name="title" content="{{title}}">
+  <meta name="description" content="{{ description }}">
 
   <!-- Open Graph / Facebook -->
   <meta property="og:type" content="website">
   <meta property="og:url" content="https://remotehack.space/">
-  <meta property="og:title" content="Remote Hack">
-  <meta property="og:description" content="{{ site.description }}">
+  <meta property="og:title" content="{{title}}">
```
Comment (lpmi-13) [inline _layouts/default.html:34]: love it

File: index.html [hunk 2]
```diff
@@ -12,38 +12,52 @@
   <div>
     {% assign sorted_hacks = site.hacks | where:"happened",false | sort:"date" | reverse %}
     {% assign next_hack = sorted_hacks | first %}
-    {% assign start_time = next_hack.date | slice:0,10 | append:"T09:30Z" %}
+    {% assign start_time = next_hack.date | slice:0,10 | append:"T08:30Z" %}
```
Comment (lpmi-13) [inline index.html:15]: I'm assuming there's some way we can make it just use UTC/GMT, and then update as needed on the client-side...but doesn't show this month anyway.

**Review by lpmi-13 — APPROVED** [review]

**Comment by benfoxall** [comment]
@spikeheap pointed out this could do with more information about what a unconf is.  Totally agree, though I'm going to merge this and try and raise another PR beefing out the info on the page!

File: index.html (hunk 2 above)
Comment (benfoxall) [reply index.html:15]: yeah, I'm not sure tbh.  We needed #89 to switch it the other way round too 😓

**Pull request merged**